bin/l2l-funall.py

```diff
@@ -45,7 +46,8 @@ def main():
         (CrossEntropyOptimizer,
          lambda: CrossEntropyParameters(pop_size=50, rho=0.2, smoothing=0.0, temp_decay=0,
                                         n_iteration=n_iterations,
-                                        distribution=NoisyGaussian(noise_decay=0.95, noise_bias=0.05))),
+                                        distribution=NoisyGaussian(noise_decay=0.95),
```

Comment:
Did you remove `noise_bias`?

Reply:
No this parameter is still there, in the previous to last line